protocol/nats_jetstream/v2/write_message.go

```diff
@@ -7,24 +7,32 @@ package nats_jetstream
 
 import (
 	"context"
+	"fmt"
+	"io"
+	"time"
+
 	"github.com/cloudevents/sdk-go/v2/binding"
 	"github.com/cloudevents/sdk-go/v2/binding/format"
-	"io"
+	"github.com/cloudevents/sdk-go/v2/binding/spec"
+	"github.com/nats-io/nats.go"
 )
 
 // WriteMsg fills the provided writer with the bindings.Message m.
 // Using context you can tweak the encoding processing (more details on binding.Write documentation).
-func WriteMsg(ctx context.Context, m binding.Message, writer io.ReaderFrom, transformers ...binding.Transformer) error {
+func WriteMsg(ctx context.Context, m binding.Message, writer io.ReaderFrom, transformers ...binding.Transformer) (nats.Header, error) {
```

Comment:
can you please update the doc string explaining how/what is returned with `nats.Header`

Reply:
also note that the returned map is not (deep) copied, which seems ok here since the message and returned values are probably short-lived...but if not 🤷‍♂️ :)